### ISSUE TITLE
VBA7 error: No App.Title or App.hInstance

### ISSUE THREAD

**Issue by fafalone** [fafalone/cTaskDialog64#6]
In `Class_Initialize` and all other relevant places,

m_sTitle should be set as:

```
#If (VBA7 <> 0) And (TWINBASIC = 0) Then
    m_stitle = "Dialog"
#Else
    m_sTitle = App.Title
#End If
```

**Comment by fafalone** [comment]
There's currently a number of App object references not properly substituted for VBA.

**Comment by fafalone** [comment]
Fixed in v1.3.7
